### DISCUSSION TITLE
Looking to Join the Slack Channel

### DISCUSSION THREAD

**Discussion by jackippolito4** [Project-Books/book-project#868] [Ideas]
Can someone link me where I can join the slack for this book project?

**Comment by knjk04** [comment]
https://join.slack.com/t/teambookproject/shared_invite/zt-punc8os7-Iz9PTCAkYcO_0S~XwtO5_A

Where did you find the Slack workspace mentioned? Perhaps we can also add a link there 

If you click on the Slack badge on the README, it should take you to an invite like for our workspace: ![image](https://user-images.githubusercontent.com/11173328/150113217-326a91d5-9715-475d-b520-bac8d72a41cf.png)
